### DISCUSSION TITLE
Recharger Issues

### DISCUSSION THREAD

**Discussion by Julu2106** [eccentricdevotion/TARDIS#745] [Q&A]
Hi,
so I started setting up a recharge point for my server. I placed the beacon and used the command "/tardisadmin recharger Shell" to set up the Recharger. But my TARDIS won't recharge while being parked right next to it. The recharger appears in the config but I can't set a perimeter for the recharging point. What did I do wrong? I added my config to this post.
[config.txt](https://github.com/eccentricdevotion/TARDIS/files/12643743/config.txt)

**Comment by eccentricdevotion** [comment]
How are you travelling to the recharge point? You must time travel, not use the `/tardis comehere` comand or the Stattenheim Remote.

**Comment by eccentricdevotion** [comment]
https://tardis.pages.dev/faqs#charging

https://tardis.pages.dev/configuration/energy/#recharge_distance

I would suggest you set up a TARDIS area around the beacon so that you can travel to it easily

https://tardis.pages.dev/tardis-areas
